api/app/wildfire_one/query_builders.py

```diff
@@ -26,7 +26,9 @@ def __init__(self):
         super().__init__()
         self.param_query = None
         # In conversation with Dana Hicks, on Apr 20, 2021 - Dana said to show active, test and project.
-        for status in ('ACTIVE', 'TEST', 'PROJECT'):
+        # Added Disabled status to fix situation where stations Disabled in WFWX API are being used in
+        # HFI Calc, causing Planning Area-level calculations to break due to missing station data.
+        for status in ('ACTIVE', 'TEST', 'PROJECT', 'DISABLED'):
```

Comment:
This is going to hang around for quite a while in classify_hfi - shouldn't we have this go into a separate PR?

Reply:
That was a residual file change from ages ago - it snuck into this PR. Removing it from the PR